### ISSUE TITLE
Usability: Embargo/Expiry

### ISSUE THREAD

**Issue by LukePercy** [silverstripe-australia/advancedworkflow#47]
Some Usability Adjustments:
1. Label each field appropriately and separately

Embargo:
'Select Publish date'

'Select Publish time'

Expiry:
'Select Un-publish date'

'Select Un-publish time'
1. Add Time Picker to time field to assist in data entry and format required.
   Currently a text field, this should probably have a time picker drop down of some sort.
2. "Click "Save Draft" to save changes" instruction.
3. Add an additional button/indication when a page is under Embargo preventing early Publishing. 
   Additionally a Content Publisher would need to be able to 'Remove' an embargo in this state - rather than editing date/time to null

Maintainer note: See framework commit 585a8bc "Removed 'showdropdown' option from TimeField, use custom libraries instead (Ingo Schommer)" This is why a dropdown no-longer shows for this workflow functionality.

**Comment by chillu** [comment]
The Terraformers (SilverStripe Ltd. team in Auckland) have started to build a separate embargo/expiry module over at https://github.com/silverstripe-terraformers/embargo-expiry. Its initial requirements are pretty basic. The main aim is to separate it from advancedworkflow. As well as getting it working with SS4 and the new `silverstripe/fluent` 2.0 version. Ideally this would work with advancedworkflow, but it's not a goal for that particular client project, so would need community help to get over the line. I'll start filing some enhancement issues in the module to map the critical path of missing features, and provide a space for more focused discussions.

**Comment by nyeholt** [comment]
Yep - ideally we'd not have two separate modules providing the same functionality, so if embargo-expiry was able to replace the functionality of advancedworkflow I'd happily remove it.
